person.rb

```diff
@@ -27,4 +28,9 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def add_rentals(rental)
+    @rental.push(rental)
+    rental.belong_to_person = self
+  end
```

Comment:
- [ ] The method should allow users to create a rental given a book and date. That's all. The rental `initialize` method will add the rental to the respective arrays.

  ```
  def add_rental(date, book)
    Rental.new(date, book, self)
  end
  ```